src/eduid_idp/loginstate.py

```diff
@@ -1,190 +0,0 @@
-#
```

Comment:
This has been moved to eduid_common.sessions, and also has been changed (reduced) to make it a session namespace dataclass. So the only way to review the changes is to compare the class as it was in eduid-IdP (below) with the one now in eduid-common.session.logindata :(

It didn't seem right to make an intermediate step with a session namespace in eduid-IdP.